tests/Integration/Behaviour/Features/Context/Domain/Product/SpecificPriceContext.php

```diff
@@ -75,9 +73,8 @@ public function transformSpecificPrice(TableNode $tableNode): SpecificPriceForEd
             PrimitiveUtils::castStringBooleanIntoBoolean($dataRows['includes tax']),
             new DecimalNumber($dataRows['price']),
             (int) $dataRows['from quantity'],
-            $from,
-            $to,
-            $this->getStoredId($dataRows, 'shop group'),
+            DateTimeUtil::NULL_DATETIME === $dataRows['from'] ? new NullDateTime() : new DateTime($dataRows['from']),
```

Comment:
Too bad I created a method for this in my multistore PR: https://github.com/PrestaShop/PrestaShop/blob/7ce88aa4f8e748c913d626915acae10adb5b38e8/src/Core/Util/DateTime/DateTime.php#L89

But it's not available yet, I went through all references anyway so I'll update all this when I will rebase it ;)

Reply:
I'll have to update it so that it returns `NullDateTime` instead of `null`